Rubrik/Private/Submit-Request.ps1

```diff
@@ -72,7 +75,7 @@ function Submit-Request {
             else {
                 if (Test-PowerShellSix) {
                     # Uses the improved ConvertFrom-Json cmdlet as provided in PowerShell 6.1
-                    $result = ConvertFrom-Json -InputObject (Invoke-RubrikWebRequest -Uri $uri -Headers $header -Method $method -Body $body)
+                    $result = ConvertFrom-Json -InputObject (Invoke-RubrikWebRequest -Uri $uri -Headers $header -Method $method -Body $body).Content
```

Comment:
We need to convert the content of the result, not the result object itself. `ExpandPayload()` does this, but it is hidden in the definition of `ExpandbyPayload()` instead of in `Submit-Request`